java/client/build.gradle

```diff
@@ -222,5 +227,6 @@ tasks.withType(Test) {
 }
 
 jar {
-    archiveBaseName = "glide"
+    archiveBaseName = "glide-for-redis"
+    archiveClassifier = "placeholder"
```

Comment:
Can you add a note please there?
Does it work locally (outside of GHA)?

Reply:
added a small comment
yes, I was not able to do a very good testing, but in theory should work.
my local build and test doesn't work